MediaStorageSample/README.md

```diff
@@ -1,15 +1,16 @@
 # Media Storage Sample
 * In this sample, you can list your Media IDs.
 
-## Setup
-* Complete [Setup](https://github.com/ricohapi/media-storage-swift#setup) of Media Storage for Swift.
+## Using the Sample
+* Clone Ricoh Media Storage for Swift by running the following commands:
+```sh
+$ git clone --recursive https://github.com/ricohapi/MediaStorage.git
+```
+* Open `MediaStorageSample.xcworkspace` in the `MediaStorageSample` directory.
```

Comment:
I think the file name is `MediaStorage.xcworkspace` and the directory is `MediaStorage`.